### DISCUSSION TITLE
中兴 B860 AV3.2M.  4+32. 写入显示成功但启动不了

### DISCUSSION THREAD

**Discussion by rodren-lion** [ophub/amlogic-s9xxx-armbian#1728] [Amlogic] [answered]
Rt
拆机看了U是  S905 L3 B. 应该是3b吧。。。。尝试过上面几个dtb 都是写入了 但无法启动 只能U盘引导。。。。索性能不能有方法dd把u盘写入呢。。。。。另外还有一个L3也是同样情况唯一不同 这个写入了 启动是能启动的 但不知道是不是系统不完整还是啥  能ping的通但ssh连显示超时。。。连不上![MVIMG_20230915_221438.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/60810175/9322d981-5691-4729-957e-92b1e2b005c0)
![IMG_20230916_203727.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/60810175/7fc6ddac-1f48-42d6-91fe-54ef3aa59fa1)

![IMG_20230916_203830.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/60810175/260bec61-2461-4712-8534-a26161f4c3e1)

![IMG_20230916_203927.jpg](https://github.com/ophub/amlogic-s9xxx-armbian/assets/60810175/1044675b-93b8-4b18-8aaa-0cfd6a53d99d)

**Comment by ophub** [comment]
https://github.com/ophub/kernel/releases/tag/tools

试试换个root了的安卓系统

**Reply by rodren-lion** [reply]
> https://github.com/ophub/kernel/releases/tag/tools
> 
> 试试换个root了的安卓系统

我再试试。。。手机几个设备都不行。。安卓都是root
的最新9了。。在adb shell 都有#

**Reply by rodren-lion** [reply, marked as answer]
已经写入了 换5.15的内核可以写入。。。6.多的太新？
写入后这货只有显示不到2.7G内存。。。1.4swap
不会是内存做交换区了吧？

**Reply by gmshiwoge** [reply]
我也有有一台同款 不知道怎么做才可以识别到4g呢

**Comment by ivantao8** [comment]
> > https://github.com/ophub/kernel/releases/tag/tools
> > 试试换个root了的安卓系统
> 
> 我再试试。。。手机几个设备都不行。。安卓都是root 的最新9了。。在adb shell 都有#

> 已经写入了 换5.15的内核可以写入。。。6.多的太新？ 写入后这货只有显示不到2.7G内存。。。1.4swap 不会是内存做交换区了吧？

tools里面没有b860的刷机包,,链个刷机包吧,,谢了.

**Comment by univerioiln233** [comment]
！居然是l3b，难怪我搞了这么久，刷不上安卓
